package.json

```diff
@@ -5,7 +5,7 @@
   "publisher": "linear",
   "author": "Linear Orbit, Inc",
   "license": "MIT",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
I think this should be changed as part of a separate commit that's tagged. eg through `npm version patch`

Reply:
Updated to have it in separate commit.